src/mpid/ch4/src/ch4_init.h

```diff
@@ -246,9 +246,6 @@ MPL_STATIC_INLINE_PREFIX int MPID_Init(int *argc,
     }
 #endif
 
-    MPIR_Process.attrs.tag_ub = (1ULL << MPIDI_CH4U_TAG_SHIFT) - 1;
```

Comment:
At the moment, the AM code does have an impact on the `tag_ub` because the AM code uses a similar `match_bits` layout as what the OFI netmod code does.

I've been working on a fix for this as part of shared memory, but there's no reason it has to stay there. I'll try to pull it out.

Reply:
That would be great.